NetCore/Resize.cs

```diff
@@ -59,6 +59,9 @@ public MagickGeometry MagickResize()
             var size = new MagickGeometry(ResizedWidth, ResizedHeight);
             using (var image = new MagickImage(MagickColor.FromRgba(0, 0, 0, 0), Width, Height))
             {
+                // 'Catrom' is generally imprecisely known as 'BiCubic' interpolation
+                image.FilterType = FilterType.Catrom;
+
```

Comment:
I'm not sure why I missed the `FilterType.Cubic` filter. Resolved in: 8fbc3c7484967f54931652623ef9e44131834992.

Note that ImageMagick seems to alias BiCubic to Catrom:
> After IM v6.7.7-7 'BiCubic' is simply an alias to 'Catrom', which is typically regarded as a good 'cubic interpolator' (b=0, c=1/2). You should use the name 'Catrom' rather than 'BiCubic' so as to be clear what you are using for interpolation.

http://www.imagemagick.org/Usage/misc/

Reply:
Ah, I see what you meant now :)